lib/teleterm/api/proto/v1/service.proto

```diff
@@ -86,6 +104,61 @@ message GetClusterRequest { string cluster_uri = 1; }
 // LogoutRequest describes LogoutRequest
 message LogoutRequest { string cluster_uri = 1; }
 
+// PasswordlessPrompt describes different prompts we need from users
+// during the passwordless login flow.
+enum PasswordlessPrompt {
+    PASSWORDLESS_PROMPT_UNSPECIFIED = 0;
+
+    // PASSWORDLESS_PROMPT_PIN is used when we need a user's pin.
+    PASSWORDLESS_PROMPT_PIN = 1;
+
+    // PASSWORDLESS_PROMPT_TAP is used when we require a user to tap their device.
+    PASSWORDLESS_PROMPT_TAP = 2;
+
+    // PASSWORDLESS_PROMPT_CREDENTIAL is used when we require a user to select a username
+    // associated with their account.
+    PASSWORDLESS_PROMPT_CREDENTIAL = 3;
```

Comment:
Thanks a lot for providing meaningful comments! 👍

Reply:
credits to @codingllama who taught me to write better comments :)